index.html

```diff
@@ -2161,8 +2199,8 @@ <h2>
         </p>
 
         <p>
-When producing and consuming DID documents that are in JSON-LD (as noted by
-the document metadata), the following rules MUST be followed.
+When producing and consuming DID documents that are in JSON-LD (as indicated by
+a <code>content-type</code> of <code>application/jsonld+did</code> in the resolver metadata), the following rules MUST be followed.
         </p>
```

Comment:
Same remark as above: <code>application/json+did</code> does not sound right. Shouldn't it be <code>application/did+json</code>?

Reply:
Correct, that was wrong, thanks @iherman!

Fixed in 4f1c79ce416dc2564e693d76cd63ede0242ccd80.